templates/products/tmpl-add-product-popup.php

```diff
@@ -177,3 +175,4 @@
         </form>
     </div>
 </script>
+<?php do_action( 'dokan_product_gallery_image_count' );?>
```

Comment:
Please change the action name `dokan_product_gallery_image_count ` to `dokan_add_product_js_template_end`